internal/upload/vmware/vmware.go

```diff
@@ -21,3 +35,22 @@ func OpenAsStreamOptimizedVmdk(imagePath string) (*os.File, error) {
 	}
 	return f, err
 }
+
+func UploadImage(creds Credentials, imagePath, imageName string) error {
+	args := []string{
+		"import.vmdk",
+		fmt.Sprintf("-u=%s:%s@%s", creds.Username, creds.Password, creds.Host),
+		"-k=true",
+		fmt.Sprintf("-pool=%s/Resources", creds.Cluster),
+		fmt.Sprintf("-dc=%s", creds.Datacenter),
+		fmt.Sprintf("-ds=%s", creds.Datastore),
+		imagePath,
+		imageName,
+	}
+	retcode := cli.Run(args)
+
+	if retcode != 0 {
+		return errors.New("importing vmdk failed")
```

Comment:
Can we somehow get a more detailed error message from `cli.Run()`?

Reply:
didn't find an option for that, `cli.Run()` always returns just the return code but it does print logs when there is a problem so more info can be found in journal

Reply:
Ah, OK... Not sure if we can do something about that in this case.

Reply:
Long term we probably want to use the underlying library directly, rather than `cli`. I suggested to start with this as we already have it working (was used in the tests), and then improve on that in a follow-up.